src/main/java/com/xebialabs/restito/semantics/Condition.java

```diff
@@ -203,6 +205,15 @@ public boolean apply(Call input) {
         });
     }
 
+    public static Condition withPostBodyContainingJsonPath(final String pattern, final String value) {
```

Comment:
if the Body is not valid Json you will receive an InvalidJsonException

if the pattern is invalid you will receive PathNotFoundException or InvalidCriteriaException depending on the issue

I could check for these exception and return false, however when developing tests it may be useful to know that you have made a mistake with your syntax.

See https://github.com/spring-projects/spring-test-mvc/blob/master/src/main/java/org/springframework/test/web/support/JsonPathExpectationsHelper.java

The spring version of this creates an AssertionError which is essentially throws an IllegalArgumentException.